README.md

```diff
@@ -2,18 +2,19 @@
+This program has several high-level goals. First, the program will survey and collate a non-exhaustive list of identity mechansisms standardized or otherwise impacted by standards developed within the IETF. Second, the program will examine existing uses of these identity mechanisms within the Internet, seeking to understand how identities are used or misused to solve real world problems. Third, the program will discuss what use cases are currently uncovered or otherwise inadequately addressed, with the intent of identifying gaps and opportunities for technical solutions to known problems that could be developed within the IETF.
 
-In working towards these goals, the program could collect important use cases addressed by existing identity mechanisms, discuss functional requirements (e.g., trust model, access control, support for roles, etc.) essential for these use cases, and showcase how existing identity mechanisms meet or exceed these requirements. The program could also look at ways in which identities are misused or misappropriated for solving problems for which there are better solutions. As an example, using client IP addresses as a form of identity may once have been a reasonable way to implement server-side security controls, but these lack uniqueness and reliability properties that are essential for meaningful security controls. 
+In working towards these goals, the program could collect important use cases addressed by existing identity mechanisms developed or influenced by IETF standards, discuss functional requirements (e.g., trust model, access control, support for roles, etc.) essential for these use cases, and showcase how existing identity mechanisms meet or exceed these requirements. Examining technologies built on IETF standards, such as [FedCM [3]](https://developer.chrome.com/en/docs/privacy-sandbox/fedcm/) and OAuth-based federated login systems, might influence these discovery tasks. The program could also look at ways in which identities are misused or misappropriated for solving problems for which there are better solutions. As an example, using client IP addresses as a form of identity may once have been a reasonable way to implement server-side security controls, but these lack uniqueness and reliability properties that are essential for meaningful security controls.
 
 Possible outputs from the program include:
```

Comment:
Do you want to add a point about creating a (non-comprehensive) taxonomy?

Reply:
I'm not sure. Do we think it would be useful?